src/service/metrics/prom.rs

```diff
@@ -534,9 +535,40 @@ pub(crate) async fn get_series(
         return Ok(vec![]);
     }
 
+    let mut sql = format!("SELECT DISTINCT({HASH_LABEL}), {label_names} FROM {metric_name}");
+    let mut sql_where = Vec::new();
+    if let Some(selector) = selector {
+        for mat in selector.matchers.matchers.iter() {
+            if mat.name == CONFIG.common.column_timestamp
+                || mat.name == NAME_LABEL
+                || mat.name == VALUE_LABEL
+            {
+                continue;
+            }
+            match &mat.op {
+                MatchOp::Equal => {
+                    sql_where.push(format!("{} = '{}'", mat.name, mat.value));
+                }
+                MatchOp::NotEqual => {
+                    sql_where.push(format!("{} != '{}'", mat.name, mat.value));
+                }
+                MatchOp::Re(_re) => {
+                    sql_where.push(format!("re_match({}, '{}'", mat.name, mat.value));
```

Comment:
Shouldn't there be a closing parenthesis?
```suggestion
                    sql_where.push(format!("re_match({}, '{}')", mat.name, mat.value));
```